cloud/passportauth/example/index.ts

```diff
@@ -1,13 +1,14 @@
-import {BunyanLogger, Logger} from '@raincatcher/logger';
+import { BunyanLogger, Logger } from '@raincatcher/logger';
```

Comment:
:+1: I tried to find a rule in tslint for this, looks like this one does it: https://palantir.github.io/tslint/rules/whitespace/ with `"check-module"`, but unfortunately doesn't seem to have an automatic fix.